.github/workflows/build_pyspinw.yml

```diff
@@ -0,0 +1,105 @@
+name: pySpinW
+
+on: [push, workflow_dispatch]
```

Comment:
Do we want to restrict it to push to `master`? I'm not sure but I think as it is it will build on a push to any branch.

Also, do we want to add a `pull_request` trigger? (Also based on `master`)? E.g.

```

on:
  push:
    branches: [main]
  pull_request:
    branches: [main]
    types: [opened, reopened, synchronize]
  workflow_dispatch:
```

Reply:
I was thinking it would be nice to get a wheel artefact for every branch. This means we could make sure pyspinw would always work

Reply:
Ok... sure.